Floofbot/Modules/Administration.cs

```diff
@@ -82,17 +95,81 @@ public class Administration : InteractiveBase
                 await Context.Channel.SendMessageAsync("⚠️ | Unable to DM user to notify them of their ban!");
             }
 
-            //bans the user
-            await Context.Guild.AddBanAsync(badUser.Id, 0, $"{Context.User.Username}#{Context.User.Discriminator} -> {reason}");
+            try
+            {
+                //bans the user
+                await Context.Guild.AddBanAsync(badUser.Id, 0, $"{Context.User.Username}#{Context.User.Discriminator} -> {reason}");
+
+                EmbedBuilder modEmbedBuilder = new EmbedBuilder();
+                modEmbedBuilder = new EmbedBuilder();
+                modEmbedBuilder.Title = (":shield: User Banned");
+                modEmbedBuilder.Color = ADMIN_COLOR;
+                modEmbedBuilder.Description = $"{badUser.Username}#{badUser.Discriminator} has been banned from {Context.Guild.Name}";
+                modEmbedBuilder.AddField("User ID", badUser.Id);
+                modEmbedBuilder.AddField("Moderator", $"{Context.User.Username}#{Context.User.Discriminator}");
+                await Context.Channel.SendMessageAsync("", false, modEmbedBuilder.Build());
+            }
+            catch 
```

Comment:
This goes for all the try/catch blocks in this PR, but the try/catch should be centered around just the `AddBanAsync` call (to avoid accidentally catching errors from `SendMessageAsync`).

Additionally, the `catch` should catch specific errors if possible, as otherwise the error information is lost. [A generic catch-all `catch` should at the very least have some way to print the serialized error type and message](https://stackoverflow.com/a/12911864).

I don't know if there's a good way to see which exceptions each function can throw (docs don't seem to be much help). [`HttpException`](https://discordnet.dev/api/Discord.Net.HttpException.html) might be helpful to intercept permissions errors, but I'd suggest looking through the docs to see what else makes sense to catch ([`RateLimitedException`](https://discordnet.dev/api/Discord.Net.RateLimitedException.html) might be another good one).

Reply:
I don't see exact specific types of exceptions. I considered adding those two aforementioned exceptions - but thought it more fitting to add a generic exception handler and updated it to spit out the exception, so it will at least catch all exceptions and spit it out. Since these are admin commands anyway - a bot admin will see the details in the appropriate mod chat anyway.

Reply:
let me know if I missed any of the exception handlers, I think I caught them all and updated, and whether you agree/disagree with my thoughts ^^

Reply:
Also added additional exception handlers to the other fun commands since some were missing